### ISSUE TITLE
[2024/01/10] Is ChatGPT Good at Search? Investigating Large Language Models as Re-Ranking Agents

### ISSUE THREAD

**Issue by zaemyung** [zaemyung/Seeking-SOTA#122]
#### Is ChatGPT Good at Search? Investigating Large Language Models as Re-Ranking Agents
- [Weiwei Sun](https://aclanthology.org/people/w/weiwei-sun/), [Lingyong Yan](https://aclanthology.org/people/l/lingyong-yan/), [Xinyu Ma](https://aclanthology.org/people/x/xinyu-ma/), [Shuaiqiang Wang](https://aclanthology.org/people/s/shuaiqiang-wang/), [Pengjie Ren](https://aclanthology.org/people/p/pengjie-ren/), [Zhumin Chen](https://aclanthology.org/people/z/zhumin-chen/), [Dawei Yin](https://aclanthology.org/people/d/dawei-yin/), [Zhaochun Ren](https://aclanthology.org/people/z/zhaochun-ren/)
- EMNLP 2023, Outstanding Paper
- https://aclanthology.org/2023.emnlp-main.923.pdf
- [rankgpt.pdf](https://github.com/zaemyung/Seeking-SOTA/files/13884682/rankgpt.pdf)
> Large Language Models (LLMs) have demonstrated remarkable zero-shot generalization across various language-related tasks, including search engines. However, existing work utilizes the generative ability of LLMs for Information Retrieval (IR) rather than direct passage ranking. The discrepancy between the pre-training objectives of LLMs and the ranking objective poses another challenge. In this paper, we first investigate generative LLMs such as ChatGPT and GPT-4 for relevance ranking in IR. Surprisingly, our experiments reveal that properly instructed LLMs can deliver competitive, even superior results to state-of-the-art supervised methods on popular IR benchmarks. Furthermore, to address concerns about data contamination of LLMs, we collect a new test set called NovelEval, based on the latest knowledge and aiming to verify the model’s ability to rank unknown knowledge. Finally, to improve efficiency in real-world applications, we delve into the potential for distilling the ranking capabilities of ChatGPT into small specialized models using a permutation distillation scheme. Our evaluation results turn out that a distilled 440M model outperforms a 3B supervised model on the BEIR benchmark. The code to reproduce our results is available at www.github.com/sunnweiwei/RankGPT.
